plugins/org.eclipse.fordiac.ide.fbtypeeditor/src/org/eclipse/fordiac/ide/fbtypeeditor/editparts/TypeEditPart.java

```diff
@@ -250,8 +256,8 @@ public INamedElement getINamedElement() {
 	@Override
 	public boolean isDirectEditable() {
 		// allow direct edit only for VarDeclarations and AdapterDeclarations
-		return super.isDirectEditable()
-				&& (getCastedModel() instanceof VarDeclaration || getCastedModel() instanceof AdapterDeclaration);
+		return super.isDirectEditable() && (getCastedModel() instanceof VarDeclaration
```

Comment:
I think now the `&& ( ... )` can be removed as a whole as it covers all interface elements.